packages/core/js-client/src/jsPeer/FluencePeer.ts

```diff
@@ -447,6 +451,8 @@ export abstract class FluencePeer {
                 !(avmCallResult instanceof Error) &&
                 avmCallResult.retCode === 0
               ) {
+                console.log(avmCallResult.callRequests, "call requiest");
+                console.log(avmCallResult.nextPeerPks, "call requiest");
```

Comment:
maybe debug log is better if you want to log something?

Reply:
thanks, i forgot to remove them